<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userupdate</class>
 <widget class="QDialog" name="userupdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New User</string>
  </property>
  <widget class="QToolButton" name="toolButton_addUser">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>230</y>
     <width>85</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD USER</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_addUserCancel">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>230</y>
     <width>85</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="Line" name="line_loweraddUser">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>215</y>
     <width>206</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_mainAddUser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_addUserTitle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>206</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ADD NEW FACCTS2 USER</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_firstNameAdd">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>80</y>
     <width>126</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_middleNameAdd">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>105</y>
     <width>126</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_lastNameAdd">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>130</y>
     <width>126</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_eMailAdd">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>170</y>
     <width>126</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_firstNameAddUser">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>80</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User - First Name*</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_middleNameAddUser">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>105</y>
     <width>126</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User - Middle Name </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_lastNameAddUser">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>130</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User - Last Name*</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_eMailAddUser">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>126</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User - Email Address*</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_addUserInstruction">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>190</y>
     <width>126</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 19);</string>
   </property>
   <property name="text">
    <string>* Required</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton_addUserCancel</sender>
   <signal>clicked()</signal>
   <receiver>userupdate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_addUser</sender>
   <signal>clicked()</signal>
   <receiver>userupdate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
